src/components/MapView/Layers/raster-utils.ts

```diff
@@ -148,7 +188,7 @@ export function WCSRequestUrl(
 
   return getWCSUrl(
```

Comment:
rename this one getWCS1Url maybe?

Reply:
Included in b4242d57a3112261cde905028318d76aac203e76